src/app/code/community/FireGento/Pdf/Model/Engine/Abstract.php

```diff
@@ -226,21 +236,20 @@ protected function Ln($height = 15)
     /**
      * Insert sender address bar
      *
-     * @param  Zend_Pdf_Page $page Current page object of Zend_Pdf
+     * @param  Zend_Pdf_Page &$page Current page object of Zend_Pdf
      *
      * @return void
      */
-    protected function _insertSenderAddessBar($page)
+    protected function _insertSenderAddessBar(&$page)
```

Comment:
Didn't you remove all the `&` references yesterday?

Reply:
Although you should not remove all:

    2017-10-14T16:28:01+00:00 ERR (3): Warning: Declaration of FireGento_Pdf_Model_Engine_Abstract::insertLogo($page, $store = NULL) should be compatible with Mage_Sales_Model_Order_Pdf_Abstract::insertLogo(&$page, $store = NULL)  in /var/www/magento193/.modman/firegento-pdf/src/app/code/community/FireGento/Pdf/Model/Engine/Abstract.php on line 0